src/core/main.cpp

```diff
@@ -293,6 +297,15 @@ int main( int argc, char * * argv )
 	qInstallMessageHandler(consoleMessageHandler);
 #endif
 
+#if defined(LMMS_HAVE_SYS_PRCTL_H) && defined(PR_SET_CHILD_SUBREAPER)
```

Comment:
This might be a dumb question but what if we don't have that corresponding header? Doesn't the fix become wasted, or is this header guaranteed to be present in the system?

Reply:
It comes with the standard C library, that is base headers like stdio.h and the like. I've taken a look at the more popular ones (glibc, musl and uClibc-ng) and they all have it. Non-Linux systems like BSDs will probably miss it, but this "child subreaper" flag is Linux-specific anyway.

Reply:
Alright then.